components/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ButtonSide from '../button/buttonSide';
+
+const Sidebar = ({ props }) => {
```

Comment:
Can you create a type for these props, please, I'm not sure about what the object props contains

Reply:
Thank you for your observation, based on the previous suggestion, I will remove the prop since that belongs to ButtonSide.